March 2020 meeting minutes (#56)
reactos/web-content: content/project-news/march-2020-meeting-minutes.md

**Colin Finck** finally launched the new website! There is likely some necessary bugfixing within the next few weeks to deal with.

**Victor Perevertkin** mostly did web stuff recently, but he said it now looks like he's ready to go on with "real things". He also tried to fix Pierre's regression about USB sticks not working.

**Mark Jansen** has been working on some smaller tasks like improving ATL and looking at tickets from Jira/GitHub. He has also been going back and forth between working on mmc_new, appcompat, some shell related projects and of course the new website. As for the future, he thinks it will be mostly the same, with some small side-projects now and then.

**Robert Naumann** started porting the Flat Remix Icon theme to ReactOS compatible ico files. He is currently searching among about 2000 icons for usable replacements for our current tango icons.

When it comes to the upcoming **ReactOS 0.4.13** release, there are no blockers from Joachim Henze's side. Currently the release is only blocked by the changelog and press release.

> Regarding **RosBE 2.2**, Victor asked if we had decided yet on selfhosting and windows 2003/XP support. Support for that is crucial to Joachim. Mark meant to look at it, but it had slipped his mind and he will create an issue to not forget about it again.

Reviewer: ```suggestion
Regarding **RosBE 2.2**, Victor asked if we had decided yet on selfhosting and Windows XP/2003 support. Support for that is crucial to Joachim. Mark meant to look at it, but it had slipped his mind and he will create an issue to not forget about it again.
```

---
title:       "March 2020 meeting minutes"
author:      "Harteex"
date:        2020-04-12
tags:        [ "meeting-minutes" ]
banner:      "img/project-news/meeting-minutes.png"
---

> 2020-01-30  

Reviewer 2: ```suggestion
2020-03-26
```

Author: Good catch!

When it comes to the upcoming **ReactOS 0.4.13** release, there are no blockers from Joachim Henze's side. Currently the release is only blocked by the changelog and press release.

Regarding **RosBE 2.2**, Victor asked if we had decided yet on selfhosting and Windows XP/2003 support. Support for that is crucial to Joachim. Mark meant to look at it, but it had slipped his mind and he will create an issue to not forget about it again.
Solutions for the other two blockers mentioned in https://jira.reactos.org/browse/CORE-16512 must also be found, especially the exception handling issue. Victor said he'd look into the exception handling. He was not sure he had enough expertise for that particular issue, but would consult Timo Kreuzer if he needed any help.

## Point 2: GSoC 2020 Status
The status of ReactOS GSoC 2020 participation was discussed. It was also noted that the GSoC timeline has shifted, in response to current events. There's now extra time for reviewing student proposals and an extended community bonding period.

## Point 3: Drop one theme from the default package

> Victor Perevertkin submitted this agenda point because he believed that the recently added themes look unfinished in many aspects and also that 4 themes is too much for the default package. He also asked about an option to make custom themes be installed from Rapps, since that has been discussed before.

Reviewer 3: ```suggestion
Victor Perevertkin submitted this agenda point because he believed that the recently added themes look unfinished in many aspects and also that 4 themes is too much for the default package. He also asked about an option to make custom themes installable from Rapps, since that has been discussed before.
```
